PNAS2009/index.html

```diff
@@ -0,0 +1,144 @@
+<!---
+layout: default
+redirect_from: "/PNAS2009.shtml"
+--->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+	<meta http-equiv="X-UA-Compatible" content="IE=10; IE=9; IE=8; IE=7; IE=EDGE">
+	<meta charset="UTF-8">
+	<link href="/images/favicon.ico" rel="shortcut icon" type="image/x-icon">
+	<title>CellProfiler | Free open-source software for measuring and analyzing cell images</title>
+	<meta property="og:title" content="CellProfiler">
+	<meta property="og:url" content="http://cellprofiler.org">
+	<meta property="og:image" content="http://cellprofiler.org/images/gallery.png">
+	<meta property="og:description" content="CellProfiler is a free open-source software for measuring and analyzing cell images">
+	<link rel="stylesheet" href="/css/app.css">
+	<!--Google Analytics-->
+	<script>
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
This is hard to read. Can you fix the formatting of this block to look more like https://github.com/CellProfiler/cellprofiler.github.com/blob/master/citations/index.html#L15-L27 ?